src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -369,8 +369,13 @@
                             </labs:SettingsCard>
                         </labs:SettingsExpander.Items>
                     </labs:SettingsExpander>
+                    <labs:SettingsCard x:Uid="GeneralPage_EnableExperimentation"
+                                       HeaderIcon="{ui:BitmapIcon Source=/Assets/FluentIcons/FluentIconsExperimentation.png}">
```

Comment:
Fluent Font icons have an icon for Insiders: `f1ad` "WindowsInsider"

Why create manual things when we can use originals?

Reply:
What does the Windows Insiders icon look like? We were intending for this icon to be a beaker to represent experimentation.